src/AppInstallerCLICore/Commands/InstallCommand.cpp

```diff
@@ -80,6 +80,7 @@ namespace AppInstaller::CLI
 
     void InstallCommand::ValidateArgumentsInternal(Execution::Args& execArgs) const
     {
+        Command::ValidateArgumentsInternal(execArgs);

```

Comment:
>Command::ValidateArgumentsInternal(execArgs); [](start = 8, length = 45)

It seems far to easy for this to happen; we should move the existing contents of Command::ValidateArgumentsInternal into Command::ValidateArguments so that it will not be overridden on accident.